n3fit/src/evolven3fit_new/evolve.py

```diff
@@ -114,6 +114,7 @@ def evolve_fit(
         info["XMax"] = float(x_grid[-1])
         # Save the PIDs in the info file in the same order as in the evolution
         info["Flavors"] = basis_rotation.flavor_basis_pids
+        info["NumFlavors"] = theory.couplings.max_num_flavs
```

Comment:
I'm currently overwriting it from eko because the eko one is wrong. When that property is removed it will already be fixed so I think it's fine.

(also, this comment answers my question on the eko PR, feel free to commit the suggestion you left there)